Subject: [PATCH] Add config files for kubernetes and aws
---
Index: modules/Dockerfile.alpine
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/modules/Dockerfile.alpine b/modules/Dockerfile.alpine
--- a/modules/Dockerfile.alpine	(revision 4b0d808b8f320df132c154a974ebe46e9e5f5ffe)
+++ b/modules/Dockerfile.alpine	(date 1695732650845)
@@ -1,6 +1,8 @@
 ARG NGINX_FROM_IMAGE=nginx:mainline-alpine
 FROM ${NGINX_FROM_IMAGE} as builder

+USER root
+
 ARG ENABLED_MODULES

 RUN set -ex \
@@ -62,6 +64,7 @@
     && echo "BUILT_MODULES=\"$BUILT_MODULES\"" > /tmp/packages/modules.env

 FROM ${NGINX_FROM_IMAGE}
+USER root
 COPY --from=builder /tmp/packages /tmp/packages
 RUN set -ex \
     && . /tmp/packages/modules.env \
@@ -69,3 +72,5 @@
            apk add --no-cache --allow-untrusted /tmp/packages/nginx-module-${module}-${NGINX_VERSION}*.apk; \
        done \
     && rm -rf /tmp/packages
+
+USER 101
\ No newline at end of file
